docs/Changelog.md

```diff
@@ -20,6 +20,7 @@
 * [#3692](https://github.com/TouK/nussknacker/pull/3692) Better logging from OpenAPI enrichers
 * [#3695](https://github.com/TouK/nussknacker/pull/3695) Fixed delaying JSON records by field in universal source
 * [#3701](https://github.com/TouK/nussknacker/pull/3701) Change: SPeL TypedMap Get Accessor
+* [#3699](https://github.com/TouK/nussknacker/pull/3699) Fix: convert json to nu struct
```

Comment:
This PR fixes not only this converter but also encoding jsons with `BestEffortJsonSchemaEncoder`, right? Maybe we should write this.

Can we also write what behaviour is beeing fixed? As I assume correctly it is, distinction between expicite null value and absence of optional field? as well in TypedMap as in encoded json?

Reply:
I've changed it to `Fix: handling null on json schema`

Reply:
My point was to provide more description what actually has changed (in particular that those two seperate things has changed). but ok :) let's leave it that way.